3rd-party/woocommerce.php

```diff
@@ -109,7 +122,8 @@ function jetpack_woocommerce_lazy_images_compat() {
 			}
 			jQuery( 'body' ).get( 0 ).dispatchEvent( jetpackLazyImagesLoadEvent );
 		} );
-	" );
+	"
+	);
```

Comment:
IMO the close-quote should line up with the open-quote on line 111.
```suggestion
		"
	);
```